examples/playground/overlays/example-append-map-to-array/_data.yml

```diff
@@ -0,0 +1,17 @@
+---
+id: 1
+type: book
+awards:
+  books:
+    - id: 1
+      title: International Botev
+      reviewers:
+        - id: 2
+          name: PersonB
+    - id: 2
+      title: Dayton Literary Peace Prize
+      reviewers:
+        - id: 3
+          name: PersonC
```

Comment:
Could you explain what we're illustrating with the first two `books` array items? I see them as similar and am wondering if it would make sense to have only one.